addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2606,6 +2606,31 @@ def test_out_invoice_reverse_move_tags(self):
             {'tax_line_id': taxes[1].id,    'tax_ids': [],              'tag_ids': tags[7].ids},
         ])
 
+    def test_out_invoice_reverse_move_account(self):
```

Comment:
Only for out_invoice ? What about in_invoice, out_refund and in_refund ? Shouldn't a version of the test be copied (and adapted) to their file each time ?